atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -135,8 +131,13 @@ def run(self):
         labels = {
             "dockerfile": self.get_pre_result(CpDockerfilePlugin.key),
             "artefacts": self.get_pre_result(DistgitFetchArtefactsPlugin.key),
-            "logs": "\n".join(self.workflow.build_logs),
-            "rpm-packages": "\n".join(self.get_post_result(PostBuildRPMqaPlugin.key)),
+
+            # We no longer store the 'docker build' logs as an annotation
+            "logs": '',
+
+            # We no longer store the rpm packages as an annotation
+            "rpm-packages": '',
```

Comment:
The idea behind this was to have compatibility with older versions of koji-containerbuild, or older osbs-client versions wanting to inspect build JSON.